modules/monitoring-enabling-the-platform-alertmanager-instance-for-user-defined-projects.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-alerts-for-user-defined-projects.adoc
+
+:_content-type: PROCEDURE
+[id="monitoring-enabling-the-platform-alertmanager-instance-for-user-defined-projects_{context}"]
+= Enabling the platform Alertmanager instance for user-defined projects
+
+You can enable user-defined alerts to use the main Alertmanager instance to fire alerts for user-defined projects. 
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have enabled monitoring for user-defined projects.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Edit the `cluster-monitoring-config` `ConfigMap` object:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
++
+. Add `enableUserAlertmanagerConfig: true` in the `alertmanagerMain` section under `data/config.yaml`:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    alertmanagerMain: 
+      enableUserAlertmanagerConfig: true <1>
+----
+<1> Set the `enableUserAlertmanagerConfig` value to `true` to allow users to create user-defined alerts that use the main platform instance of Alertmanager.
++
+. Save the file to apply the changes. The main Alertmanager instance restarts automatically and applies the new configuration.
```

Comment:
I'm not sure that the Alertmanager pods are going to be restarted.

Reply:
not restarted